src/main/java/matchingAlgorithm/SimpleUnitCapacityRequestInserterModule.java

```diff
@@ -38,7 +38,7 @@ protected void configureQSim() {
 						maxEuclideanDistance)))
 				.asEagerSingleton();
 
-		bindModal(VehicleAssignmentTools.class).toProvider(new ModalProviders.AbstractProvider<>(drtCfg.getMode()) {
+		/*bindModal(VehicleAssignmentTools.class).toProvider(new ModalProviders.AbstractProvider<>(drtCfg.getMode()) {
```

Comment:
No uncomented code please. Either remove it or keep it. 

I suspect that you didn't want to remove it, because this was written by somebody else.